src/components/Markdown/Markdown.tsx

```diff
@@ -115,6 +117,17 @@ export const Markdown: FunctionComponent<{
           return `https://${githubPrefix}/${owner}/${repo}/${githubSuffix}/${url}`;
         };
 
+  const byteLength = Buffer.byteLength(children);
+  if (byteLength > ONE_MEGABYTE) {
+    console.log(`Doc page is larger than 1MB, showing only README...`);
```

Comment:
I guess it's a debug message that we should remove?

Reply:
Removed it.